packages/cli/src/commands/labs/index.ts

```diff
@@ -0,0 +1,87 @@
+import color from '@heroku-cli/color'
+import {Command, flags} from '@heroku-cli/command'
+import * as Heroku from '@heroku-cli/schema'
+import {Args, ux} from '@oclif/core'
+import {sortBy} from 'lodash'
+
+interface Features {
+  currentUser: any,
+  user: any,
+  app?: any,
+}
+
+function printJSON(features: Heroku.Account | Heroku.AccountFeature | Heroku.AppFeature) {
+  ux.log(JSON.stringify(features, null, 2))
+}
+
+function printFeatures(features: any) {
+  features = sortBy(features, 'name')
+  const longest = Math.max.apply(null, features.map((f: Record<string, string>) => f.name.length))
```

Comment:
Although this is legacy code, there may be opportunities to update how these `Math` functions are called.
```suggestion
  const longest = Math.max(...features.map((f: Record<string, string>) => f.name.length))
```
In this case, a spread operator suffices and shortens the call expression.